R/05-crispr-calc.R

```diff
@@ -107,32 +81,37 @@ calc_crispr <- function(.data = NULL,
         target_type == "gene_ctrl" ~ gRNA1_seq,
         target_type == "ctrl_gene" ~ gRNA2_seq
       ),
+      control_gRNA_seq = case_when(
```

Comment:
Some of these steps had to be rearranged. @kweav and I went through this together to figure it out. Needed the double control crisprs to be aligned with the single gene targets they are related to.